src/ReportManager/DetailsSection/SchemaForm/fields/Text/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { useContext } from 'react';
+
+import useFieldDetails from '../../SchemaFormContext/useFieldDetails';
+import { SchemaFormContext } from '../../SchemaFormContext';
+import { isFieldRequired } from '../../utils';
+
+import styles from './styles.module.scss';
+
+const ShortTextInput = ({ textFieldDetails, fieldName, onChange, className }) => (
+  <input id={fieldName}
+           type="text"
+           value={textFieldDetails.value || textFieldDetails.defaultInput}
+           placeholder={textFieldDetails.placeholder}
+           onChange={onChange}
+           data-testid={`schema-form-short-text-field-input-${fieldName}`}
+           className={className} />
+);
+
+const LongTextInput = ({ textFieldDetails, fieldName, onChange, className }) => (
+  <textarea id={fieldName}
+            value={textFieldDetails.value || textFieldDetails.defaultInput}
+            placeholder={textFieldDetails.placeholder}
+            onChange={onChange}
+            data-testid={`schema-form-long-text-field-input-${fieldName}`}
+            className={className}>
+  </textarea>
+);
+
+const INPUT_TYPE = {
+  SHORT: 'SHORT_TEXT',
+  LONG: 'LONG_TEXT'
+};
+
+const TEXT_INPUT_TYPE_TO_INPUT = {
+  [INPUT_TYPE.SHORT]: ShortTextInput,
+  [INPUT_TYPE.LONG]: LongTextInput,
+};
+
+const VALIDATION_ERROR_TYPES = {
+  REQUIRED: 'required'
+};
+
+export const TextFieldValidators = {
```

Comment:
Remember we won't be using these validators, we should remove this and any related code.

Reply:
Sure